simulator/src/test/java/com/hazelcast/simulator/probes/impl/HdrProbeTest.java

```diff
@@ -26,16 +30,19 @@ public void testConstructor_noThroughputProbe() {
         assertFalse(tmpProbe.isPartOfTotalThroughput());
     }
 
+    // this test is fragile and can easily fail if there is some kind of pausing going on
```

Comment:
Your solution seems to be "more fragile", since it checks for an exact value, not a range? Or did I misread the new test?